build/testing/integration.go

```diff
@@ -43,10 +44,17 @@ func init() {
 	}
 }
 
+func max(a, b int) int {
+	if a > b {
+		return a
+	}
+	return b
+}
+
```

Comment:
```suggestion
```

I think we could use [built-in max func](https://go.dev/ref/spec#Min_and_max) here